script/panes/shares.js

```diff
@@ -103,9 +103,10 @@ module.exports = class SharesPane extends Pane {
                     for (var i = 0; i < widget.organizations.length; i++) {
                         var org = widget.organizations[i];
                         org.element = _self.createListItem(org);
-    
                         list.appendChild(org.element);
                     }
+                    document.querySelector(".tooltiptext2").style.top = "40px" // Moves the highest tooltip down
```

Comment:
Bruk heller :nth-child i CSS :)

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child

Reply:
Har tittet på :nth-child og :nth-of-type() men klarer ikke finne en løsning med disse, siden hver tooltiptext ligger inni en egen <div class="tooltip> som har hover funksjonen, fordi jeg ikke fikk lagt hover direkte på <a>-taggen med bildet.